### PR TITLE
Bump version to 0.4.3-rc1

### PR DESCRIPTION
**Pull request by kchibisov**
This includes some bugfixes, especially some that are critical for Windows.

### CODE REVIEW

File: .builds/freebsd.yml [hunk 1]
```diff
@@ -12,10 +12,10 @@ sources:
 tasks:
   - rustup: |
       curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable --profile minimal
-  - 1-39-0: |
-      $HOME/.cargo/bin/rustup toolchain install --profile minimal 1.39.0
+  - 1-37-0: |
+      $HOME/.cargo/bin/rustup toolchain install --profile minimal 1.37.0
       cd alacritty
-      $HOME/.cargo/bin/cargo +1.39.0 test
+      $HOME/.cargo/bin/cargo +1.37.0 test
```
Comment (chrisduerr) [inline .builds/freebsd.yml:18]: Why are we downgrading minimum version here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
...
+## 0.4.3
+
+### Fixed
+
+- Fallback to `LC_CTYPE=UTF-8` on macOS without valid system locale
+- Emoji colors blending with terminal background
+- Fix escapes prematurely terminated by terminators in unicode glyphs
+- Incorrect location when clicking inside an unfocused window on macOS
+- Startup mode `Maximized` on Windows
+- Crashing on start on Windows
```
Comment (chrisduerr) [inline CHANGELOG.md:16]:
```suggestion
- Crashing at startup on Windows
```

File: alacritty/Cargo.toml [hunk 3]
```diff
@@ -22,7 +22,7 @@ glutin = { version = "0.24.0", features = ["serde"] }
 notify = "4"
 libc = "0.2"
 unicode-width = "0.1"
-parking_lot = "0.9"
+parking_lot = "0.10.2"
```
Comment (chrisduerr) [inline alacritty/Cargo.toml:25]: Is it necessary to bump this? Does this benefit us in any way?

File: alacritty/Cargo.toml [hunk 4]
```diff
@@ -34,7 +34,7 @@ rustc_tools_util = "0.2.0"
 xdg = "2"
 
 [target.'cfg(not(target_os = "macos"))'.dependencies]
-image = { version = "0.22.3", default-features = false, features = ["ico"] }
+image = { version = "0.23.3", default-features = false, features = ["ico"] }
```
Comment (chrisduerr) [inline alacritty/Cargo.toml:37]: Is it necessary to bump this? Does this benefit us in any way?

File: alacritty_terminal/Cargo.toml [hunk 5]
```diff
@@ -13,10 +13,10 @@ libc = "0.2"
 notify = "4"
 bitflags = "1"
 font = { path = "../font" }
-parking_lot = "0.9"
+parking_lot = "0.10.2"
```
Comment (chrisduerr) [inline alacritty_terminal/Cargo.toml:16]: Is it necessary to bump this? Does this benefit us in any way?

File: alacritty_terminal/Cargo.toml [hunk 6]
```diff
@@ -27,7 +27,7 @@ url = "2"
 copypasta = { version = "0.6.3", default-features = false }
 
 [target.'cfg(unix)'.dependencies]
-nix = "0.16.1"
+nix = "0.17.0"
```
Comment (chrisduerr) [inline alacritty_terminal/Cargo.toml:30]: Is it necessary to bump this? Does this benefit us in any way?

File: font/Cargo.toml [hunk 7]
```diff
@@ -17,10 +17,10 @@ servo-fontconfig = "0.4.0"
...
 
 [target.'cfg(target_os = "macos")'.dependencies]
-core-foundation = "0.6"
-core-text = "13"
-core-graphics = "0.17"
-core-foundation-sys = "0.6"
+core-foundation = "0.7"
+core-text = "15"
+core-graphics = "0.19"
+core-foundation-sys = "0.7"
```
Comment (chrisduerr) [inline font/Cargo.toml:23]: Is it necessary to bump this? Does this benefit us in any way?

**Comment by kchibisov** [comment]
done

**Comment by damnskippy** [comment]
Windows font fallback https://github.com/alacritty/alacritty/pull/3127 was a big (i.e. impactful) addition for windows.  Perhaps should be listed in the notes.  I believe this is the first release that includes the PR, IIRC.

**Comment by chrisduerr** [comment]
>  I believe this is the first release that includes the PR.

Version 0.4.3 does not include Windows font fallback.

**Comment by damnskippy** [comment]
Thanks, my bad.

**Pull request merged**